packages/default-theme/components/account/MyAddresses/Address.vue

```diff
@@ -0,0 +1,178 @@
+<template>
+<div v-if="address">
+  <div class="shipping__content">
+  <p class="shipping__address">
+    <span class="shipping__client-name">
+      {{ firstName }} {{ lastName }}
+    </span>
+    <br />
```

Comment:
Please, don't use `<br>` tag

Reply:
you can make it on your own ;) I just took it from Ui example so it's always better to have it done than perfect :+1: